modules/fbx_importer/SCsub

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+Import("env")
+Import("env_modules")
+
+env_assimp = env_modules.Clone()
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/include"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/code/Importer/IFC"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/code/FBX"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/misc"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/code"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/common"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/contrib/irrXML/"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/contrib/unzip/"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/code/Importer/STEPParser"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/zlib/"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/contrib/openddlparser/include"])
+env_assimp.Prepend(CPPPATH=["#thirdparty/assimp/contrib/rapidjson/include"])
+env_assimp.Prepend(CPPPATH=["."])
+env_assimp.Prepend(CPPPATH=["data/"])
+env_assimp.Prepend(CPPPATH=["tests/"])
+# env_assimp.Append(CPPDEFINES=['ASSIMP_DOUBLE_PRECISION']) # TODO default to what godot is compiled with for future double support
+env_assimp.Append(CPPDEFINES=["ASSIMP_BUILD_BOOST_WORKAROUND"])
+env_assimp.Append(CPPDEFINES=["OPENDDLPARSER_BUILD"])
+env_assimp.Append(CPPDEFINES=["ASSIMP_BUILD_NO_OWN_ZLIB"])
+env_assimp.Append(CPPDEFINES=["ASSIMP_BUILD_NO_EXPORT"])
+env_assimp.Append(CPPDEFINES=["ASSIMP_BUILD_NO_X_IMPORTER"])
```

Comment:
Are these still necessary?

Reply:
yup

Reply:
I will strip out all of assimp except the FBX folder in a bit, maybe once we deliver next milestone.